### DISCUSSION TITLE
Possible to install without MS Store/choco?

### DISCUSSION THREAD

**Discussion by luciofstars** [File-New-Project/EarTrumpet#1591] [Q&A] [answered]
I'm installing Windows on my laptop (*sigh*) to use Creative Cloud and this app seemed interesting, **However**, I use LTSC, which guts the Store and its related components, Can I install EarTrumpet? (preferably without Chocolatey, since that's not great either,)

**Comment by riverar** [comment, marked as answer]
Some installation methods are listed in the [readme](https://github.com/File-New-Project/EarTrumpet?tab=readme-ov-file#install). But you can also just [pull down the package](https://github.com/microsoft/winget-pkgs/blob/master/manifests/f/File-New-Project/EarTrumpet/2.3.0.0/File-New-Project.EarTrumpet.installer.yaml#L10) and open it with 7-Zip or similar.

I'm actively rebuilding the build/distribution pipeline so I'll make sure to consider integration with GitHub Releases.
